### PR TITLE
[UPD] delivery_package_fee: delivery fee by stock package type

### PR DESCRIPTION
**Pull request by nguyenminhchien**

### CODE REVIEW

File: delivery_package_fee/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Delivery Package Fees",
     "summary": "Add fees on delivered packages on shipping methods",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.1.0",
```
Comment (simahawk) [inline delivery_package_fee/__manifest__.py]: This is not needed unless you have migration steps.
Otherwise the bot will take care.

Comment (nguyenminhchien) [reply delivery_package_fee/__manifest__.py]: updated, thanks.

**Review by cyrilmanuel — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by simahawk — APPROVED** [review]

**Pull request merged**